include/sot/core/admittance-control-op-point.hh

```diff
@@ -16,13 +16,13 @@
 /* --------------------------------------------------------------------- */
 
 #if defined(WIN32)
-#  if defined(admittance_control_op_point_EXPORTS)
-#    define ADMITTANCECONTROLOPPOINT_EXPORT __declspec(dllexport)
-#  else
-#    define ADMITTANCECONTROLOPPOINT_EXPORT __declspec(dllimport)
-#  endif
+#if defined(admittance_control_op_point_EXPORTS)
+#define ADMITTANCECONTROLOPPOINT_EXPORT __declspec(dllexport)
 #else
-#  define ADMITTANCECONTROLOPPOINT_EXPORT
+#define ADMITTANCECONTROLOPPOINT_EXPORT __declspec(dllimport)
+#endif
+#else
+#define ADMITTANCECONTROLOPPOINT_EXPORT
```

Comment:
Is there a reason no to use `SOT_CORE_DLLAPI` provided by the jrl CMake module ? I know a lot of files in this repo do not use it.

Reply:
This is for windows support, right ? 

In this case, is anybody actually using it ? Human, or CI ?
If not, we can consider that it doesn't work, and just remove those useless instructions.

Reply:
I will have a trainee who will probably have to work on it.
But for now this is indeed not useful.
And indeed it would be better to use SOT_CORE_DLLAPI

Reply:
I suggest to leave the corrections for another PR